src/palace/manager/core/coverage.py

```diff
@@ -1334,7 +1334,10 @@ def _set_circulationdata(self, identifier, circulationdata):
         """
         error = None
         if circulationdata:
-            primary_identifier = circulationdata.primary_identifier(self._db)
+            try:
```

Comment:
Do you want to keep this exception in here long-term? Or is it just to make sure that it's behaving as you think it is in the short term? If the latter, would be handy to put a "TODO:" in here to note that.